src/findersync.cs

```diff
@@ -54,6 +72,21 @@ interface FIFinderSyncProtocol
 
 		[Export ("toolbarItemToolTip")]
 		string ToolbarItemToolTip { get; }
+
+	    [Export ("supportedMessageInterfaceNamesForItemWithURL:")]
+	    string[] GetSupportedMessageInterfaceNamesForItem (NSUrl itemURL);
```

Comment:
Same: `itemUrl`

Also I think `GetSupportedMessageInterfaceNames` would be simpler and still won't lose information.